exporters/prometheus/config.go

```diff
@@ -20,12 +20,15 @@ import (
 
 // config contains options for the exporter.
 type config struct {
-	registerer prometheus.Registerer
+	registerer       prometheus.Registerer
+	createTargetInfo bool
```

Comment:
Redefining this as `disableTargetInfo` will allow using a default value of `false`. Having the default the same as the zero value of a `bool` helps avoid having to set this here and in tests.

Reply:
yup, I did it like this to make it easier to read on the `getMetricData()` method (which imo is the most important code path). 

I do prefer `if c.createTargetInfo` than `if !c.disableTargetInfo` but no strong feelings about it, will update.

Reply:
Updated.